start-mailserver.sh

```diff
@@ -188,6 +188,22 @@ echo "required_score 5" >> /etc/mail/spamassassin/local.cf
 echo "rewrite_header Subject ***SPAM***" >> /etc/mail/spamassassin/local.cf
 cp /tmp/spamassassin/rules.cf /etc/spamassassin/
 
+
+echo "Configuring fail2ban"
+# enable filters
+perl -i -0pe 's/(\[postfix\]\n\n).*\n/\1enabled  = true\n/'     /etc/fail2ban/jail.conf
```

Comment:
At first I also tried sed. But to replace multiline strings with perl is much easier :)

See https://unix.stackexchange.com/questions/26284/how-can-i-use-sed-to-replace-a-multi-line-string for example.